docs/lightning/04-salesforce-lambdas/04-contact-channel-analytics.md

```diff
@@ -168,9 +168,9 @@ Please note that your secret may also be formatted stored as a "Secret key/value
 
 2. For the **Name** and **Label**, enter AwsGenerateAudioRecordingURL.
 
-3. In the **URL** section, enter `https://lambda.{awsRegion}.amazonaws.com/2015-03-31/functions/{lambdaFunctionName}/invocations/`-- replace {awsRegion} with the awsRegion your serverless application resides in (for example, us-east-1), and replace {lambdaFunctionName} with the full name of the sfGenerateAudioRecordingStreaming lambda you recorded in the previous section.
+3. In the **URL** section, enter `https://lambda.{awsRegion}.amazonaws.com/2015-03-31/functions/{lambdaFunctionName}/invocations`-- replace {awsRegion} with the awsRegion your serverless application resides in (for example, us-east-1), and replace {lambdaFunctionName} with the full name of the sfGenerateAudioRecordingStreaming lambda you recorded in the previous section.
```

Comment:
We need to remove the trailing slash (/) after "invocations" in the URL under SFDC Named Credentials. In order to import the call recording streaming on SFDC side.

Reply:
At one point not including the `/` causes callouts to fail (see [here](https://salesforce.stackexchange.com/a/279151)), now in some instances including the `/` causes the call the call to fail... In the end we instead chose to keep the `/` in the url but add a note discussing the issue - see [here](https://github.com/amazon-connect/amazon-connect-salesforce-cti/blob/main/docs/lightning/04-salesforce-lambdas/04-contact-channel-analytics.md#salesforce-side-setup)

Reply:
@blee-amzn yeah sounds good. I agree with the note.